src/ipc/ipc-zephyr.c

```diff
@@ -69,6 +71,7 @@ static bool message_handler(const struct device *dev, void *arg, uint32_t data,
 	increment_ipc_received_counter();
 #endif
 
+	pm_policy_state_lock_get(PM_STATE_RUNTIME_IDLE, PM_ALL_SUBSTATES);
```

Comment:
could you explain how the power state transition was possible before this commit?

Reply:
ack - @tmleman I think we should spell this out as an inline comment so that no one is tempted to change PM state during IPC processing. Is suspect we can only change PM state when entering idle ?